loader-tests/README.md

```diff
@@ -4,16 +4,13 @@ Integration tests for each social network or news format.
 
 This project allows testing of different types of web feed aggregators on the Internet and provides assurance that our reader can load tests from real feeds.
 
-## Currently supported formats:
+## Check Loaders OPML files
```

Comment:
We are planning to have many different tests here, so I gave a description of specific test